frontend/src/components/utils/NavBar/index.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { ReactElement, useState, useEffect } from 'react';
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  makeStyles,
+  Button,
+  IconButton,
+  Drawer,
+  Link,
+  MenuItem,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import { Link as RouterLink } from 'react-router-dom';
+import icon from '../../../assets/home-icon.png';
+
+export type navbarButton = {
```

Comment:
nit: best practice for types is to capitalize each first letter of every word in the type name